### ISSUE TITLE
Caught AttributeError when openning first file

### ISSUE THREAD

**Issue by Timandes** [seanliang/ConvertToUTF8#68]
After disabling and enabling the plugin, when I opened some file, I got an error in console:
```
Traceback (most recent call last):
  File "C:\Users\名字\AppData\Roaming\Sublime Text 3\Packages\ConvertToUTF8\ConvertToUTF8.py", line 50, in save_on_dirty
    sublime.set_timeout(self.save_on_dirty, 10000)
AttributeError: 'NoneType' object has no attribute 'set_timeout'
```

Then the plugin seems stucked and does not convert any file until I disable an enable it again.

Could you guide me to solve that?

**Comment by seanliang** [comment]
This bug is fixed in v1.2.11, please update the plugin and have check. Thanks!

**Comment by Timandes** [comment]
It works fine. Thanks.
